nmoscommon/webapi.py

```diff
@@ -603,9 +610,27 @@ def __init__(self, oauth_config=None):
 
         self.add_routes(self, basepath='')
 
-        # Enable ProxyFix middleware if required
+        # If the `fix_proxy` option is "enabled", interpret it as "use one proxy"
         if _config.get('fix_proxy') == 'enabled':
-            self.app.wsgi_app = ProxyFix(self.app.wsgi_app)
+            if HAS_WERKZEUG_MIDDLEWARE:
+                self.app.wsgi_app = ProxyFix(self.app.wsgi_app, x_for=1, x_proto=1, x_host=1, x_port=1, x_prefix=1)
+            else:
+                self.app.wsgi_app = ProxyFix(self.app.wsgi_app)
+        # If the `fix_proxy` option is a number, interpret it as the number of proxies to trust (Werkzeug>=0.15.0)
```

Comment:
Would you mind updating https://github.com/bbc/nmos-common#configuration to reflect the use of this variable? I note it says 'boolean' at the moment which should really be a string.